control-flow/includes/7-knowledge-check.md

```diff
@@ -0,0 +1,32 @@
+# Knowledge check
```

Comment:
Is it possible for you to write a short explanation for each one of these answers? Whether they are correct or not, the questions are all supposed to have a sentence explaining why it's correct or not. 

It would be really helpful. Let me know if that's too much trouble.

Reply:
no worries, I've included the explanations in parenthesis, let me know